src/twisted/newsfragments/7930.bugfix

```diff
@@ -0,0 +1,2 @@
+Raise CannotDetermineNamespaceError instead of KeyError when unable to
```

Comment:
As per the examples in https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles this should refer to a fully-qualified python name; what is it that raises the exception?